game/pokemon.py

```diff
@@ -6,6 +6,8 @@
 
 
 class Pokemon:
+    """Holds information about Pokemon in our own format"""
```

Comment:
Our own format could be anything we define it to be, so I think that bit is kind of redundant. Maybe describe how it holds the 'current instance' of the Pokemon, since it only exists for as long as the Pokemon is in the player's party or in battle as an enemy.

Reply:
Changed to `Holds information about Pokemon in current instance of the application`

Reply:
Minor: since the word 'Pokemon' can refer to one or several Pokemon, consider changing this to refer to 'a Pokemon`

Reply:
Changed to `Holds information about a Pokemon in current instance of the application`